webpack/JobInvocationDetail/JobInvocationConstants.js

```diff
@@ -16,10 +16,12 @@ export const CANCEL_RECURRING_LOGIC = 'CANCEL_RECURRING_LOGIC';
 export const GET_REPORT_TEMPLATES = 'GET_REPORT_TEMPLATES';
 export const GET_REPORT_TEMPLATE_INPUTS = 'GET_REPORT_TEMPLATE_INPUTS';
 export const JOB_INVOCATION_HOSTS = 'JOB_INVOCATION_HOSTS';
+export const GET_TEMPLATE_INVOCATION = 'GET_TEMPLATE_INVOCATION';
+export const MAX_HOSTS_API_SIZE = 'MAX_HOSTS_API_SIZE';
```

Comment:
This should probably be a number since `hostsToOpen.slice(0, MAX_HOSTS_API_SIZE)` doesnt work for a string